app/controllers/administrative-units/administrative-unit/sites/site/edit.js

```diff
@@ -0,0 +1,76 @@
+import Controller from '@ember/controller';
+import { dropTask } from 'ember-concurrency';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class AdministrativeUnitsAdministrativeUnitSitesSiteEditController extends Controller {
+  @service router;
+  @tracked isPrimarySite =
+    this.site.get('id') === this.administrativeUnit.get('primarySite.id');
```

Comment:
Wait, does this work? :smile:  The controller shouldn't have access to these values when it's created and it won't rerun when new data arrives through the model hook.

Changing this into a getter should be work for sure.

Reply:
Ah wait, I see now, it's also used to store the checkbox value. Hmmm. I think it makes sense to have easy access to the value before it was edited.

Maybe adding an `isPrimaryCurrently` or something better named might be nice?